indico/modules/users/controllers.py

```diff
@@ -297,35 +303,78 @@ def _process(self):
                 if isinstance(entry, User):
-                    search_results.append(UserEntry(profile_url=url_for('.user_profile', entry),
-                                                    **{k: getattr(entry, k) for k in IDENTITY_ATTRIBUTES}))
+                    search_results.append(UserEntry(
+                        profile_url=url_for('.user_profile', entry),
+                        **{k: getattr(entry, k) for k in IDENTITY_ATTRIBUTES}
+                    ))
                 else:
                     search_results.append(UserEntry(
-                        profile_url=None, full_name="{first_name} {last_name}".format(**entry.data.to_dict()),
-                        **{k: entry.data.get(k) for k in (IDENTITY_ATTRIBUTES - {'full_name'})}))
-
+                        profile_url=None,
+                        full_name="{first_name} {last_name}".format(**entry.data.to_dict()),
+                        **{k: entry.data.get(k) for k in (IDENTITY_ATTRIBUTES - {'full_name'})}
+                    ))
             search_results.sort(key=attrgetter('first_name', 'last_name'))
+
+        num_reg_requests = RegistrationRequest.query.count()
         return WPUsersAdmin.render_template('users_admin.html', form=form, search_results=search_results,
-                                            num_of_users=num_of_users, num_deleted_users=num_deleted_users)
+                                            num_of_users=num_of_users, num_deleted_users=num_deleted_users,
+                                            num_reg_requests=num_reg_requests)
+
+
+class RHUsersAdminSettings(RHAdminBase):
+    """Manage global suer-related settings."""
+
+    def _process(self):
+        form = AdminUserSettingsForm(obj=FormDefaults(**user_management_settings.get_all()))
+        if form.validate_on_submit():
+            user_management_settings.set_multi(form.data)
+            return jsonify_data(flash=False)
+        return jsonify_form(form)
 
 
 class RHUsersAdminCreate(RHAdminBase):
     """Create user (admin)"""
 
+    CSRF_ENABLED = True
+
+    def _create_user(self, form):
+        user = User()
+        user.first_name = form.first_name.data
+        user.last_name = form.last_name.data
+        user.email = form.email.data
+        user.affiliation = form.affiliation.data
+        identity = Identity(provider='indico', identifier=form.username.data, password=form.password.data)
+        user.identities.add(identity)
+        user.favorite_users.add(user)
+        db.session.add(user)
+        db.session.flush()
+        signals.users.registered.send(user, from_moderation=True, identity=identity)
+        return user
+
     def _process(self):
-        form = LocalRegistrationForm()
-        return WPUsersAdmin.render_template('users_create.html', form=form)
+        form = AdminAccountRegistrationForm()
+        if form.validate_on_submit():
+            user = self._create_user(form)
+            msg = Markup('{} <a href="{}">{}</a>').format(
```

Comment:
If you want to re-order the arguments here you simply change the function call. But for i18n you might have different orders in different languages.